docs/getting-started/index.asciidoc

```diff
@@ -5,5 +5,6 @@
 
 Looking to get started with Elastic Security? This section describes the Elastic Security UI in Kibana, the system requirements required to run the Elastic Agent with the Elastic Endpoint Security integration, as well as instructions on how to configure and install Elastic Security on your host.
 
-include::siem-ui.asciidoc[]
+include::security-ui.asciidoc[]
+include::network-page-overview.asciidoc[]
```

Comment:
This change solved the following error for me:
> ERROR: getting-started/network-page-overview.asciidoc: line 2: invalid part, must have at least one section (e.g., chapter, appendix, etc.)
If that's not the nesting you want, however, let me know and we can play with other options.
```suggestion
include::network-page-overview.asciidoc[leveloffset=+1]
```